test/src/tests-audience-manager.ts

```diff
@@ -21,7 +17,7 @@ declare global {
     }
 }
 
-const userAudienceUrl = `https://${Constants.DefaultBaseUrls.userAudienceUrl}${apiKey}/audience?mpid=${testMPID}`
+const userAudienceUrl = `https://${Constants.DefaultBaseUrls.userAudienceUrl}${apiKey}/audience?mpid=`
```

Comment:
I think this is a broader issue and probably should be its own ticket as part of the api client refactor, which i made here https://mparticle-eng.atlassian.net/browse/SQDSDKS-6234.  2 reasons.  

1. Our URLs have a few things in common, but when we use something like Helpers.createServiceUrl, there can still be a bunch of appending things still happening.

https://github.com/mParticle/mparticle-web-sdk/blob/4740596cb9f19e968e74c4d3c103d50c846d0888/src/identityApiClient.js#L115-L131

2. In a test environment, I think there is value in knowing exactly what format a URL is, as opposed to sending it to a function.

Do you have a specific suggestion, or is my interpretation of your comment off?

Reply:
1. You bring up some interesting points and I think as part of the ticket you created, we probably should write unit tests around url generation logic specifically so that it things are better organized.

2. I agree with this completely